src/libraries/System.Text.Json/src/Resources/Strings.resx

```diff
@@ -717,4 +717,7 @@
   <data name="InvalidIndentSize" xml:space="preserve">
     <value>Indentation size must be between {0} and {1}.</value>
   </data>
+  <data name="InvalidNewLine" xml:space="preserve">
+    <value>Only the strings '\n' or '\r\n' are permitted.</value>
```

Comment:
It's kind of odd to see single quotes delimit string literals, but OTOH I've never seen double quotes used in exception messages either.

Reply:
Yeah I wasn't quite sure what exactly to use here, but many of the other messages in this file quote with single quotes. Happy to reword if there's a better way to express it.

Reply:
@bartonjs might have opinions on this

Reply:
While we often use single quotes for things other than System.Char, it generally seems to always be for things other than strings.  (Often chars (from parsers), occasionally enum values, or property names, etc).

When it comes to actual strings, though, one can find examples both ways.  To me this feels closest to `guid.ToString(format)` rejecting anything except a few limited inputs... which uses double-quotes:

https://github.com/dotnet/runtime/blob/839c642c3abebe5ba7fcd18b5b03dee50dc1b76e/src/libraries/System.Private.CoreLib/src/Resources/Strings.resx#L2401-L2406